index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+
+    <body class="container">
+        <header>
+            <div class="header">
+                <div class="logo">
+                   <a href=""><img src="My Logo.png" alt="" class="logo-img" /></a> 
+                </div>
+                <div class="menu">
+                    <img src="Union.svg" alt="">
+                </div>
```

Comment:
Well done overall  💯  but the hamburger button should have a `nav` tag.
   * The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.